articles/mysql/flexible-server/how-to-restore-dropped-server.md

```diff
@@ -29,7 +29,7 @@ To restore a deleted Azure Database for MySQL flexible server instance, you need
 1. In the Activity Log, select **Add filter** as shown and set the following filters for the
 
     - **Subscription** = Your Subscription hosting the deleted server
-    - **Resource Type** = Azure Database for MySQL flexible servers (Microsoft.DBforMySQL/flexibleServers)
+    - **Resource Type** = Azure Database for MySQL Flexible Servers (Microsoft.DBforMySQL/flexibleServers)
```

Comment:
Note that the instance in line 32 is plural, so it's not clear whether this refers to the product name (which should be capitalized) or to an instance, which might not need to be capitalized.